test/CMakeLists.txt

```diff
@@ -15,11 +15,6 @@ if (NOT TARGET gtest OR NOT TARGET gmock)
 	endif ()
 endif ()
 
-#add_executable(${PROJECT_NAME}_example ${CMAKE_CURRENT_SOURCE_DIR}/test.cpp)
-#target_link_libraries(${PROJECT_NAME}_example PRIVATE ${PROJECT_NAME})
-#set_target_properties(${PROJECT_NAME}_example PROPERTIES FOLDER tests CMAKE_CXX_STANDARD_REQUIRED ON CXX_EXTENSIONS OFF)
-#enable_lto(${PROJECT_NAME}_example)
-
```

Comment:
why remove these lines? the test example was just fine.
If you really want to get rid of the CMake target, then also remove the corresponding file from the repository.

Reply:
I don't think I have added any test.cpp file.

Reply:
Interesting. Then I really don't know where this was coming from in the beginning. Consider this resolved then.